client/app/projects/projects.controller.js

```diff
@@ -1,15 +1,30 @@
 'use strict';
 
 angular.module('observatory3App')
-.controller('ProjectsCtrl', function ($scope, $location, $http) {
+
+.controller('ProjectsCtrl', function ($scope, $location, $http, Auth) {
     $scope.projects = [];
     $scope.projectToAdd = {active: true};
+    $scope.loggedIn = false; 
+
+    Auth.isLoggedInAsync(function(loggedIn){
+        if (loggedIn){
+            var user = Auth.getCurrentUser();
+            $scope.user = user;
+            $scope.checkUserProject();
+        }
+    });
 
     $scope.getCurrentProjects = function() {
         $http.get('/api/projects').success(function(projects) {
             $scope.projects = projects;
         });
         $scope.past = false;
+        Auth.isLoggedInAsync(function(loggedIn){
```

Comment:
Why do this again? Why not just set the loggedIn flag in the other function that you call on page load?